### ISSUE TITLE
Opening hcdp issues to FundRequest

### ISSUE THREAD

**Issue by andreafno** [HydroCommunity/Community-Brainstorming#25]
Hi,
could be an idea integrating hdcp issues with [FundRequest](https://fundrequest.io/)? I think this could boost development, and you can pay developers with HYDRO tokens if I understood well.

Happy Halloween

**Comment by thelandoftheliving** [comment]
Looks like a great idea!

**Comment by andreafno** [comment]
I talked with an admin, he confirmed that 
> If the token is erc20 then we’d only need to authorise it in our system.

Developers can be payed directly in HYDRO.

**Comment by MasterSensei** [comment]
Looks interesting. Do they take a commission?

**Comment by andreafno** [comment]
Answer by Tim Dierckxsens:

> Hey Andrea, it is still free to use, we don’t take a commission. If they use the platform funded issues will be posted on FundRequest and on their GitHub repository. We push notifications signalling they have an active bounty and advise whether the bounty is I) clear and II) funded appropriately.

**Comment by AnuragHydro** [comment]
Personally, long-term I see Snowflake as providing a better solution for this (see [#30](https://github.com/HydroCommunity/Community-Brainstorming/issues/30))

**Comment by andreafno** [comment]
I agree for long-term, but opening the project to another platform (now) IMO could be a good way to reach more developers that doesn't know about Hydro yet.
